crates/octez/src/async/bootstrap.rs

```diff
@@ -209,18 +209,140 @@ impl BootstrapContracts {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub enum SmartRollupPvmKind {
+    #[serde(rename = "wasm_2_0_0")]
+    Wasm,
+    #[serde(rename = "arith")]
+    Arith,
```

Comment:
This is not a real PVM. It was a reference implementation when the pvm code first came into being. Can be removed